### PR TITLE
support alluxio container network

### PR DESCRIPTION
**Pull request by allenhaozi**
Signed-off-by: mahao <allenhaozi@gmail.com>

### Ⅰ. Describe what this PR does
support alluxio container network

### Ⅱ. Does this pull request fix one issue?

#989 

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Comment by allenhaozi** [comment]
@cheyang Please review this pr. thank you

File: cmd/webhook/app/init.go [hunk 1]
```diff
@@ -23,6 +23,6 @@ func NewWebHookCommand() *cobra.Command {
 		Use:   "fluid-webhook",
 		Short: "fluid admission webhook",
 	}
-	cmd.AddCommand(versionCmd, webhookCmd)
+	cmd.AddCommand(versionCmd, cmd)
```
Comment (zwwhdls) [inline cmd/webhook/app/init.go:26]: Why change to `cmd` ?

File: cmd/webhook/app/webhook.go [hunk 2]
```diff
@@ -17,26 +17,27 @@ package app
 
 import (
 	"flag"
-	"github.com/fluid-cloudnative/fluid/pkg/common"
```
Comment (cheyang) [inline cmd/webhook/app/webhook.go:20]: I think there is no need to change this file. Thanks.

File: charts/alluxio/values.yaml [hunk 3]
```diff
@@ -12,7 +12,6 @@
 # This should not be modified in the usual case.
 fullnameOverride: alluxio
```
Comment (cheyang) [inline charts/alluxio/values.yaml:14]: Could you update the version and changelog in alluxio chart? thanks.

File: charts/alluxio/templates/fuse/daemonset.yaml [hunk 4]
```diff
@@ -12,6 +12,7 @@
 {{ if .Values.fuse.enabled -}}
 {{- $shortCircuitEnabled := .Values.shortCircuit.enabled -}}
 {{- $needDomainSocketVolume := and $shortCircuitEnabled (eq .Values.shortCircuit.policy "uuid") }}
+{{- $fuseHostNetwork := .Values.fuse.hostNetwork }}
 
 apiVersion: apps/v1
```
Comment (cheyang) [inline charts/alluxio/templates/fuse/daemonset.yaml:17]: Do we need  the following configuration?

```yaml
 - name: ALLUXIO_WORKER_CONTAINER_HOSTNAME
            valueFrom:
              fieldRef:
                fieldPath: status.podIP
```

**Review by cheyang — APPROVED** [review]

**Pull request merged**